pom.xml

```diff
@@ -246,6 +246,11 @@
         <artifactId>aws-java-sdk-minimal</artifactId>
         <version>1.12.246-349.v96b_b_f7eb_a_c3c</version>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>1.7.36</version>
```

Comment:
This dep is managed for all plugins. (This particular version is included in core as of https://github.com/jenkinsci/jenkins/pull/6263.) If you were getting a `RequireUpperBoundDeps` error about it, use an `<exclusion>` instead—matching reality (the plugin will actually run with whatever version core bundles). https://github.com/jenkinsci/artifact-manager-s3-plugin/blob/744d17d3dd2fc77704ef93cac832900149f2c163/src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/MinioIntegrationTest.java#L72 is responsible for verifying that this all actually works at runtime.

Reply:
> This dep is managed for all plugins. (This particular version is included in core as of [jenkinsci/jenkins#6263](https://github.com/jenkinsci/jenkins/pull/6263).) If you were getting a `RequireUpperBoundDeps` error about it, use an `<exclusion>` instead—matching reality (the plugin will actually run with whatever version core bundles).
> 
> https://github.com/jenkinsci/artifact-manager-s3-plugin/blob/744d17d3dd2fc77704ef93cac832900149f2c163/src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/MinioIntegrationTest.java#L72
> 
> is responsible for verifying that this all actually works at runtime.

Fixed